moveit_core/constraint_samplers/include/moveit/constraint_samplers/default_constraint_samplers.h

```diff
@@ -69,6 +69,18 @@ class JointConstraintSampler : public ConstraintSampler
   JointConstraintSampler(const planning_scene::PlanningSceneConstPtr& scene, const std::string& group_name)
     : ConstraintSampler(scene, group_name)
   {
+    int rng_seed;
+    if (ros::param::get("~constraint_sampler_random_seed", rng_seed))
+    {
+      ROS_WARN_STREAM_NAMED("constraint_samplers",
```

Comment:
Why did you decided for a warning here? This is an info at most, preferably a debug message, isn't it?
```suggestion
      ROS_INFO_STREAM_NAMED("constraint_samplers",
```

Reply:
Well, I had a bias from a production view. Generally, debug messages are not enabled in normal runs, which may cause this to slip away from the eyes. Anyways a simple rosparam lookup would still give the information needed, thus reducing to debug 👍🏼